Dockerfile

```diff
@@ -19,39 +18,32 @@ RUN bin/build-ui.sh
 
 
 # Next, all the things for building the Python web service
-
 FROM python:3.7-slim@sha256:3e4be41076ebb6fe8c3112b220ce133ef0dc49c814024e4874ca76eae3c8dec0
 
+WORKDIR /app/
```

Comment:
Moving this earlier will make sure the directory is created and usable and that everything happens in that cwd.

Reply:
Is this a different directory compared to that on line 15 (from the Node part)?

Reply:
I'm not really sure what you mean by that.

This part of the docker file created a workdir in /tmp and then switched back to /app and it had a mkdir call. So I replaced that with this line.